keymanapp/keyman
# chore(windows): remove WM_UNICHAR completely

Fixes: #8677 

For this PR we can just load two different keyman keyboards and test that characters are still being output

## User Testing

# TEST_TAMIL_OUTPUT

Install Tamil 99 keyboard
*  Open Notepad, select the Tamil Keyboard 
* Type in some characters and ensure they are output
* Open MS `Word` if you have it otherwise, Firefox
* Verify that typing produces the expected output

# TEST_KHMER_ANGKOR

Install Khmer Angkor keyboard

*  Open Notepad, select the Khmer_Angkor Keyboard 
* Type in some characters and ensure they are output
* Open MS `Word` if you have it otherwise, Firefox
* Verify that typing produces the expected output
